README.md

```diff
@@ -1,20 +1,26 @@
-Code: [![Version](https://badge.fury.io/rb/noid-rails.png)](http://badge.fury.io/rb/noid-rails)
+# Noid::Rails
```

Comment:
It looks like this is missing the section on who the product owner is and what it means to be a Core Component.

Reply:
Down on line 23